assets/js/front/epoch-front-compiled.js

```diff
@@ -147,7 +147,7 @@ jQuery( document ).ready( function ( $ ) {
              *
              * @since 0.0.1
              */
-            $( app.form_el ).submit( function( event ) {
+            $( document ).on( 'submit', '#' + epoch_vars.form_id, function( event ) {
```

Comment:
Strange. I didn't intend to change that. Maybe one was outdated?
On Mon, Aug 31, 2015 at 22:58 Josh Pollock <notifications@github.com> wrote:

> In assets/js/front/epoch-front-compiled.js
> <https://github.com/postmatic/epoch/pull/64#discussion_r38383864>:
>
> > @@ -147,7 +147,7 @@ jQuery( document ).ready( function ( $ ) {
> >               *
> >               * @since 0.0.1
> >               */
> > -            $( app.form_el ).submit( function( event ) {
> > +            $( document ).on( 'submit', '#' + epoch_vars.form_id, function( event ) {
>
> Can I ask why switching to on() instead of submit() here? Also makes me
> nervous binding this to document, instead of the specific element, since we
> may be dealing with two different documents when in iFrame mode.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/postmatic/epoch/pull/64/files#r38383864>.
>